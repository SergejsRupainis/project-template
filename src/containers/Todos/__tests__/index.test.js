import React from 'react';
import { shallow, mount } from 'enzyme';

import { Todos } from '../index';

describe('<Todos />', () => {
  it('should render title', () => {
    const title = 'My dummy title';
    const props = {
      changePage: jest.fn(),
      fetchTodos: jest.fn(),
      refresh: jest.fn(),
      todos: [],
      isFething: false,
      title,
      lastUpdated: 1546003385534,
      error: '',
    };

    const wrapper = shallow(<Todos {...props} />);
    expect(wrapper.find('h1').text()).toBe(title);
    expect(wrapper).toMatchSnapshot();
  });
});
