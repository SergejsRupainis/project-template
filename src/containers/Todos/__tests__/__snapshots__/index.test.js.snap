// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todos /> should render title 1`] = `
<div>
  <h1>
    My dummy title
  </h1>
  <h2>
    Empty.
  </h2>
  <div>
    Last updated at 
    15:23:05
    .
     
  </div>
  <button
    onClick={[Function]}
    type="button"
  >
    Refresh
  </button>
  <button
    onClick={[Function]}
    type="button"
  >
    Go to about page via redux
  </button>
</div>
`;
